### PR TITLE
[FLOC-2921] Release Flocker 1.2.0

### PR DESCRIPTION
**Pull request by dwgebler**
FLOC-2921

### CODE REVIEW

**Comment by dwgebler** [comment]
Awaiting build results...

File: docs/releasenotes/index.rst [hunk 1]
```diff
@@ -13,6 +13,8 @@ You can learn more about where we might be going with future releases by:
 v1.2
 ====
 
+* If you upgrade to Docker 1.8.1 you may find pulling images unreliable in flocker-deploy and the Flocker Containers API due to Docker bug #15699.
```
Comment (adamtheturtle) [inline docs/releasenotes/index.rst]: :question: Link to the Docker issue?

**Comment by adamtheturtle** [comment]
Assuming that you are happy with the build results when GitHub is back, continue.

**Comment by adamtheturtle** [comment]
Please continue

**Comment by dwgebler** [comment]
Finally, a successful build!

**Pull request merged**